scripts/src/javatests/org/oppia/android/scripts/maven/MavenDependenciesListWriterTest.kt

```diff
@@ -0,0 +1,338 @@
+          dependenciesList.add(
+            getMavenDependency(
+              artifactName = "com.github.bumptech.glide:annotations:4.11.0",
+              version = "4.11.0"
+            )
+          )
+        DependencyName.IO_FABRIC ->
+          dependenciesList.add(
+            getMavenDependency(
+              artifactName = "io.fabric.sdk.android:fabric:1.4.7",
+              version = "1.4.7"
+            )
+          )
+        else -> dependenciesList.add(
+          getMavenDependency(
+            artifactName = "androidx.databinding:databinding-adapters:3.4.2",
+            version = "3.4.2"
+          )
+        )
+      }
+    }
+    return MavenDependencyList
+      .newBuilder()
+      .addAllMavenDependency(dependenciesList)
+      .build()
+  }
+
+  private fun getLicense(
+    licenseName: String,
+    primaryLink: String = "",
+    primaryLinkType: PrimaryLinkType = PrimaryLinkType.UNSPECIFIED,
+    alternativeLink: String = ""
+  ): License {
+    return License
+      .newBuilder()
+      .setLicenseName(licenseName)
+      .setPrimaryLink(primaryLink)
+      .setPrimaryLinkType(primaryLinkType)
+      .setAlternativeLink(alternativeLink)
+      .build()
+  }
+
+  private fun writeMavenInstallJson(file: File) {
+    file.writeText(
+      """
+      {
+        "dependency_tree": {
+          "dependencies": [
+            {
+              "coord": "androidx.databinding:databinding-adapters:3.4.2",
+              "url": "https://maven.google.com/androidx/databinding/databinding-adapters/3.4.2/databinding-adapters-3.4.2.aar"
+            },
+            {
+              "coord": "com.github.bumptech.glide:annotations:4.11.0",
+              "url": "https://repo1.maven.org/maven2/com/github/bumptech/glide/annotations/4.11.0/annotations-4.11.0.jar"
+            },
+            {
+              "coord": "com.google.firebase:firebase-analytics:17.5.0",
+              "url": "https://maven.google.com/com/google/firebase/firebase-analytics/17.5.0/firebase-analytics-17.5.0.aar"
+            },
+            {
+                "coord": "com.google.protobuf:protobuf-lite:3.0.0",
+                "url": "https://repo1.maven.org/maven2/com/google/protobuf/protobuf-lite/3.0.0/protobuf-lite-3.0.0.jar"
+            },
+            {
+              "coord": "io.fabric.sdk.android:fabric:1.4.7",
+              "url": "https://maven.google.com/io/fabric/sdk/android/fabric/1.4.7/fabric-1.4.7.aar"
+            }
+          ]
+        }
+      }  
+      """.trimIndent()
+    )
+  }
+
+  private enum class DependencyLicenseType {
+    NO_LICENSES,
+    PLAIN_TEXT_AND_SCRAPABLE_LICENSE,
+    PLAIN_TEXT_AND_NONSCRAPABLE_LICENSE,
+    NON_PLAIN_TEXT_LICENSE,
+    INVALID_LINK_LICENSE;
+  }
+
+  private enum class DependencyName {
+    GLIDE_ANNOTATIONS,
+    FIREBASE_ANALYTICS,
+    DATA_BINDING,
+    PROTO_LITE,
+    IO_FABRIC;
+  }
+
+  private fun initializeMockNetworkAndBazelUtils(): NetworkAndBazelUtils {
```

Comment:
add kdoc

Reply:
it was no more needed and hence removed.

Reply:
@prayutsu I think this hasn't been resolved yet. PTAL.

Reply:
Done.